package.json

```diff
@@ -4,33 +4,42 @@
   "description": "User Interface for team FlashFeet",
   "main": "app.js",
   "scripts": {
-    "start": "npx tsc && cp -R views dist && node dist/app.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint ."
+    "start": "npx tsc && cp -R views dist && cp -r public dist && node dist/app.js",
+    "test": "mocha test/unit/service/AuthServiceTests.js",
+    "lint": "eslint .",
+    "test-ui": "mocha --timeout 10000 test/ui/**/*.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "@actions/core": "^1.10.1",
     "@types/express": "^4.17.21",
-    "@types/express-session": "^1.17.9",
+    "@types/express-session": "^1.17.10",
     "@types/node": "^20.10.1",
-    "axios": "^1.3.3",
+    "axios": "^1.6.2",
+    "axios-mock-adapter": "^1.22.0",
```

Comment:
Couple of these packages are devDependencies. e.g. "@types/express-session": "^1.17.10", "axios-mock-adapter": "^1.22.0", "@types/node": "^20.10.1", ...